src/main/scala/com/yoo/app/ImageDatabaseApp.scala

```diff
@@ -61,6 +61,19 @@ class ImageDatabaseApp(imageDao: ImageDAO, disk: DiskService)(implicit ec: Execu
     })
   }
 
+  get("/images/:id") {
+    if (FeatureConfig.SAVE_IMAGE_TO_DB) {
```

Comment:
Feature toggling. If `true` the endpoint for retrieving an image and its base64 encoded representation is enabled.